test/express/middleware-test.js

```diff
@@ -20,6 +20,19 @@ describe('input-validation middleware tests - Express', function () {
             middleware.init('test/pet-store-swagger.yaml');
         });
     });
+    describe('initAsync function tests', async function () {
+        it('should throw an error in case the file doesn\'t exists', function () {
+            const middleware = require('../../src/middleware');
+            expect(async () => {
```

Comment:
Are you sure that this works the way you think it does? Considering that you are running async code inside the sync block, are you sure it will execute before the test completes?
Since chai does not support something similar to expect.assertions in Jest, I would suggest adding some kind of flag setting to ensure that the code in question was executed or do the try-catch block to be on the safe side.

Reply:
modified to used try-catch and assert